php/simps/config/servers.php

```diff
@@ -19,8 +19,6 @@
         ],
         'settings' => [
             'worker_num' => swoole_cpu_num() * 2,
-            'open_tcp_nodelay' => true,
-            'enable_reuse_port' => true,
```

Comment:
why removing this ? I suppose the performance decrease for this use case ?

Reply:
I feel like it's useless because I only start one port.

Reply:
reuse_port is about port sharing (same port)
https://lwn.net/Articles/542629/

btw, it could be useless here, honestly I don't know how `swoole` works

any informations @limingxinleo @huangzhhui ?

Reply:
`enable_reuse_port` is useless for this point.

I think it could be removed from project.

Reply:
Is it not enabled by default ? I mean should we not set it to `false` ?

Reply:
The default is false, so there is no need to set

Reply:
ok